src/components/Footer/index.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { P, H3 } from '@/styles/text';
+import {
+  FooterContainer,
+  FooterSection,
+  FooterDiv,
+  IconDiv,
+  FooterLink,
+} from './styles';
+import Icon from '../Icon';
+
+export default function Footer() {
+  return (
+    <FooterContainer>
+      <FooterDiv>
+        <FooterSection>
+          <Image
+            alt="footer image"
+            src="/footerimage.webp"
+            placeholder="blur"
+            blurDataURL="/footerimage.webp"
+            quality={100}
+            width={168}
+            height={66}
+          />
+          <hr style={{ width: '160px' }} />
+          <P $color="white">2727 Camino del Rio S</P>
+          <P $color="white">©2024 by Immigration Justice Project Pro Bono</P>
+        </FooterSection>
+      </FooterDiv>
```

Comment:
should we add this blurb here?
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/fbb71e8b-ae47-46d0-8cfd-e268576ebe0f)